src/tools/features-status-dump/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use std::collections::HashMap;
+use std::io::BufWriter;
+use std::{env, fs::File};
+use std::path::Path;
+use tidy::features::{collect_lang_features, collect_lib_features, Feature};
+
+#[derive(serde::Serialize)]
+struct FeaturesStatus {
+    lang_features_status: HashMap<String, Feature>,
+    lib_features_status: HashMap<String, Feature>,
+}
+
+fn main() {
+    let library_path_str = env::args_os().nth(1).expect("library/ path required");
+    let compiler_path_str = env::args_os().nth(2).expect("compiler/ path required");
+    let output_path_str = env::args_os().nth(3).expect("output path required");
```

Comment:
We might be able to extend `x.py` to either set these automatically, or provide default paths here.

Reply:
Suggestion: what you can do is that in the bootstrap build step, pass these paths (which bootstrap will know about) as cli flags to this tool.

Reply:
See https://github.com/rust-lang/rust/pull/133514#issuecomment-2505191223